src/status_im/chat/models/message_content.cljs

```diff
@@ -13,19 +13,21 @@
                         [:tag     constants/regx-tag]
                         [:mention constants/regx-mention]])
 
+(def blank " ")
```

Comment:
Would you mind explaining why we need to do this? (annotate with `^js`) thanks

Reply:
annotations are replacing externs.js, when something is annotated as ^js it prevents the advanced compilation from minifying the function call. Here I couldn't annotate `" "` so I put it in a def

Reply:
I probably went overkill in some cases it's not needed

Reply:
It improves the Feng shui of the file.

Reply:
now, that's a good reason